ncm-nss/src/main/perl/nss.pm

```diff
@@ -113,6 +113,10 @@ sub Configure {
             my $nscdrestart = CAF::Process->new(
                 ['service', 'nscd', 'condrestart'], log => $self);
             $nscdrestart->execute();
+	    # wait for nscd to restart before returning in case
```

Comment:
are you aware of https://github.com/quattor/configuration-modules-core/blob/master/ncm-authconfig/src/main/perl/authconfig.pm#L379? it's not condrestart, but for sure does not assume this a simple restart will just work.